.github/workflows/tests.yml

```
@@ -356,6 +356,19 @@ jobs:
           HOMEBREW_BUILDPULSE_ACCOUNT_ID: 1503512
           HOMEBREW_BUILDPULSE_REPOSITORY_ID: 53238813
 
+      - id: junit_xml
+        run: |
+          cd "${{ steps.set-up-homebrew.outputs.repository-path }}"
+          filenames=$(find test/junit -name 'rspec*.xml' -print | tr '\n' ',')
+          echo "filenames=${filenames%,}" >> "$GITHUB_OUTPUT"
+
+      - uses: codecov/test-results-action@v1
```

We pin the other `codecov/codecov-action` so do we want to pin this one too?

```suggestion
      - uses: codecov/test-results-action@9739113ad922ea0a9abb4b2c0f8bf6a4aa8ef820 # v1.0.1
```